src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+.mainheader {
+  font-size: 17px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.articleheader {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
+
+.link {
+  color: green;
+}
+
+.paragraph {
+  margin: 0;
+}
```

```suggestion
.mainHeader,
.paragraph {
  margin: 0;
}
```
Don't repeat yourself. You use the rule `margin: 0` two times - both in your `.mainHeader` and `.paragraph` selectors. You may write it like I've shown above and remove `margin: 0` from your `.mainHeader`